frontend/pages/dashboard.tsx

```diff
@@ -1,9 +1,21 @@
 import { LoanIcon, LoanPlusIcon, PersonPlusIcon } from '@/icons';
 import DashboardLayout from '@/Layouts/DashboardLayout';
 import Analytics from '@/modules/dashboard/components/Analytics';
-import { ReactNode } from 'react';
+import { ReactNode, useEffect } from 'react';
+import { useSession, signIn, signOut } from 'next-auth/react';
+import { useRouter } from 'next/router';
+
 
 const Dashboard = () => {
```

Comment:
I think this redirection should be done using the getServerSideProps

<img width="841" alt="image" src="https://github.com/harunurkst/qard-al-hasana/assets/36256353/1b0a8877-268f-4ea0-a49b-a54ea67935d4">